pkgs/development/python-modules/deepdiff/default.nix

```diff
@@ -14,16 +15,14 @@
 
 buildPythonPackage rec {
   pname = "deepdiff";
-  version = "6.2.1";
+  version = "6.2.3";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
-  src = fetchFromGitHub {
-    owner = "seperman";
-    repo = "deepdiff";
-    rev = "refs/tags/${version}";
-    hash = "sha256-AKah3A9srKm/cFWM7IiZ7JxQ8s0KTuh8VLKOymsDgnA=";
+  src = fetchPypi {
```

Comment:
Why not stick with `fetchFromGitHub`?

Reply:
Oh, I thought the most recent versions weren't tagged on GitHub, and it looks like I was wrong.